packages/components/src/components/typography/typography.stories.tsx

```diff
@@ -24,49 +24,49 @@ export const AllTypography: StoryFn = () => (
                 feugiat vel mi.
             </Paragraph>
             <Paragraph typographyStyle="titleLarge" data-testid="paragraph-titleLarge">
-                type="titleLarge" <br />
+                type=&#34;titleLarge&#34; <br />
```

Comment:
I think this is better readable (and all other `&#34;` ocurrences as wel)
```suggestion
                type=&quot;titleLarge&quot; <br />
```

Reply:
fixed: https://github.com/trezor/trezor-suite/pull/15287/commits/e4bbc4687be479cc752e1243ebefbed1000a899b